UnityProject/Assets/Scripts/Health/Objects/Integrity.cs

```diff
@@ -133,8 +134,8 @@ private void EnsureInit()
 		if (registerTile != null) return;
 		if (SMALL_BURNING_PREFAB == null)
 		{
-			SMALL_BURNING_PREFAB = Resources.Load<GameObject>("SmallBurning");
-			LARGE_BURNING_PREFAB = Resources.Load<GameObject>("LargeBurning");
+			SMALL_BURNING_PREFAB = Resources.Load<GameObject>("BurningSmall");
```

Comment:
these ``Resources.Load`` could be replaced with another way of referencing the prefabs, such as a serialized field.

Reply:
Gave it a go but would have to update every prefab that uses the component.

Reply:
Then move the reference to a static like the CommonPrefabs